src/App.jsx

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
+import { Sum } from './components/Sum';
+
 import './App.scss';
 
 const App = () => (
-  <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
-  </>
+  <ul className="list">
+    <li className="list__item">
```

Comment:
You don't have 'list__item' class
```suggestion
    <li className="list__item">
```